src/vmm/src/builder.rs

```diff
@@ -57,9 +57,7 @@ use crate::vmm_config::boot_source::BootConfig;
 use crate::vmm_config::drive::BlockDeviceType;
 use crate::vmm_config::instance_info::InstanceInfo;
 use crate::vmm_config::machine_config::{MachineConfigUpdate, VmConfig, VmConfigError};
-use crate::vstate::memory::{
-    create_memfd, GuestAddress, GuestMemory, GuestMemoryExtension, GuestMemoryMmap,
```

Comment:
Do we still want to keep `create_memfd()` pub?

Reply:
Good point, lets make it private

Reply:
Done!